hm_pyhelper/miner_param.py

```diff
@@ -0,0 +1,48 @@
+import os
+import subprocess
+import logging
+import json
+
+
+def get_public_keys_rust():
+    """
+    Run gateway_mfr and report back the key.
+    """
+    direct_path = os.path.dirname(os.path.abspath(__file__))
+    gateway_mfr_path = os.path.join(direct_path, 'gateway_mfr')
+
+    try:
+        run_gateway_mfr_keys = subprocess.run(
+            [gateway_mfr_path, "key", "0"],
+            capture_output=True,
+            check=True
+        )
+    except subprocess.CalledProcessError:
+        logging.error("gateway_mfr exited with a non-zero status")
+        return False
+
+    try:
+        return json.loads(run_gateway_mfr_keys.stdout)
+    except json.JSONDecodeError:
+        logging.error("Unable to parse JSON from gateway_mfr")
+    return False
+
+
+def get_mac_address(path):
```

Comment:
@ilyastrodubtsev can you bring in the other helper function that calls on this function?